cypress/support/parent-commands.ts

```diff
@@ -0,0 +1,168 @@
+     */
+    checkPageAside(): void;
+
+    /**
+     * Asserts that:
+     * - at least one live preview is in page
+     */
+    checkLivePreviews(): void;
+
+    /**
+     * Asserts that:
+     * - at least one do/don't container is in page
+     */
+    checkDoDonts(): void;
+
+    /**
+     * Asserts that:
+     * - change log container is visible
+     * - change log header `h2` with text "Changlog" is visible
+     * - on click of header, the changelog content is visible.
+     */
+    checkChangelogRevealer(): void;
+  }
+}
+
+Cypress.Commands.add('getDocsPageContentArea', () => cy.get('#paste-docs-content-area'));
+
+Cypress.Commands.add('pageHeaderShouldBeVisible', (headerText) => {
+  cy.contains('h1', headerText).shouldBeVisible();
+});
+
+Cypress.Commands.add('overviewTableRendersCorrectly', () => {
+  cy.get('table').first().as('componentsTable').should('be.visible');
+  cy.get('@componentsTable').find('thead').find('tr').find('th').as('overviewTableHeaders');
+  cy.get('@componentsTable').find('tbody').find('tr').as('tableRows');
+
+  // Table has correct number of headers
+  cy.get('@overviewTableHeaders').should('have.length', 6);
+
+  // Table has correct text for headers
+  const headers = ['Name', 'Status', 'Code ready', 'Design assets', 'Documentation', 'Peer review'];
+  cy.get('@overviewTableHeaders').each((header, index) => {
+    // using include here to do a loose match of headers
+    cy.wrap(header).should('include.text', headers[index]);
+  });
+
+  // verify at least one row is rendered.
+  cy.get('@tableRows').its('length').should('be.greaterThan', 0);
+});
+
+Cypress.Commands.add('checkInPageNavigationLinks', () => {
+  cy.getDocsPageContentArea().as('contentArea');
+
+  cy.get('@contentArea').find('[data-cy="page-aside-anchor"]').as('pageAsideAnchors');
+
+  cy.get('@contentArea')
+    .find(
+      '[data-cy="anchored-heading-h2"]:not(#component-changelog a),[data-cy="anchored-heading-h3"]:not(#component-changelog a)'
+    )
+    .as('anchoredHeadings');
+
+  cy.get('@pageAsideAnchors').then((anchors) => {
+    cy.get('@anchoredHeadings').each((anchor, idx) => {
+      cy.wrap(anchor).should('have.attr', 'href').and('include', '#');
+      cy.wrap(anchors[idx]).should('have.attr', 'href').and('include', '#').and('eql', anchor.attr('href'));
+    });
+  });
+});
+
+// @TODO Check ComponentHeader <--- waiting for changes to this component to be merged.
+// Cypress.Commands.add('checkComponentHeader', () => {});
+
+Cypress.Commands.add('checkPageAside', () => {
+  cy.getDocsPageContentArea().getInFixedContainer('[data-cy="page-aside"]').as('pageAside');
+  cy.get('@pageAside').find('[data-cy="table-of-contents"]').shouldBeVisible();
+  cy.get('@pageAside').contains('button', 'Rate this page').shouldBeVisible();
+  cy.get('@pageAside').find('[data-cy="page-aside-anchor"]').shouldBeVisible();
+});
+
+Cypress.Commands.add('checkLivePreviews', () => {
```

Comment:
Some JSDocs on all of these would be really handy, this seems like the meat and potatos file.

Reply:
There are JSDocs above, with the TS declarations.

Reply:
If you look at the branch in an editor you should see the intellisense.